js/lib/DateFmt.js

```diff
@@ -842,14 +851,16 @@ DateFmt.prototype = {
     },
 
     // stand-alone of m (month) is l
+    // stand-alone of my (month year) is mys
     // stand-alone of d (day) is a
     // stand-alone of w (weekday) is e
     // stand-alone of y (year) is r
     _standAlones: {
-        "l": "m",
-        "a": "d",
-        "e": "w",
-        "r": "y"
+        "m": "l",
+        "my": "mys",
+        "d": "a",
```

Comment:
oh I see, standAlone for `my` is newly added. :)  It's pretty much clean up. Regarding the format, `a` Isn't it ok to be removed? It look like there's no format data and test case too.